docs/pages/agents/core-messaging/create-a-client.mdx

```diff
@@ -108,10 +108,11 @@ type ClientOptions = {
    * Add a client app version identifier that's included with API requests.
    * Production apps are strongly encouraged to set this value.
    *
-   * For example, you can use the following format: `appVersion: 'APP_NAME/APP_VERSION'`.
+   * You can use the following format: `appVersion: 'AGENT_NAME-agent/AGENT_VERSION'`.
+   * For example, `appVersion: 'alix-agent/2.x'`
```

Comment:
WDYT of also adding an app example, like `appVersion: "APP_NAME-app/APP_VERSION"`?

maybe we should say something along the lines of "if you have an agent and an app, it's best to distinguish them from each other." but if a dev only has 1 or the other, the `-agent` and `-app` suffixes aren't really needed.